### PR TITLE
(SIMP-6508) Install rsync skeleton into /usr

### PR DESCRIPTION
**Pull request by jeannegreulich**
- moved the installation of rsync skeleton files into
  /usr/share/simp/environments.  This is to prevent
  the update of operational files.

SIMP-6508 #close

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,19 +55,26 @@ module itself, you need to perform the following operations:
...
-#### Copy the Repository into Place
+#### Copy the Repository to the skeleton directory
 
-  1. ``mkdir -p /var/simp/environments/simp``
-  2. Copy the ``environments/simp/rsync`` directory from this repository into ``/var/simp/environments/simp``
-  3. ``cd /var/simp/environments/simp/rsync``
+  1. ``mkdir -p /usr/share/simp/environments``
+  2. Copy the ``rsync`` directory from this repository into ``/usr/share/simp/environments``
+  3. ``cd /usr/share/simp/environments/rsync``
   4. ``setfacl --restore=.rsync.facl 2>/dev/null``
```
Comment (lnemsick-simp) [inline README.md]: Is this setfacl necessary in /usr/share/simp?

File: README.md [hunk 2]
```diff
@@ -53,19 +55,26 @@ module itself, you need to perform the following operations:
...
-  1. ``cd`` into the cloned repository
-  2. ``cd build/selinux``
-  3. ``make -f /usr/share/selinux/devel/Makefile``
-  4. ``cp *.pp /usr/share/selinux/packages``
-  5. ``/usr/sbin/semodule -n -i /usr/share/selinux/packages/simp-rsync.pp``
-  6. ``/usr/sbin/load_policy``
-  7. ``/sbin/fixfiles -R simp-rsync restore
+ simp-environments-skeleton contains the selinux profile.
+ The selinux profile was moved to simp-environments to because
+ the two profiles were oversriting each other.
```
Comment (lnemsick-simp) [inline README.md]: oversriting -> overwriting

What does this mean?

File: Rakefile [hunk 3]
```diff
@@ -4,7 +4,7 @@ require 'simp/rake'
 
 class RsyncPkg < Simp::Rake::Pkg
   def define_clamsync
-    clambase = 'environments/simp/rsync/Global/clamav'
+    clambase = 'rsync/Global/clamav'
```
Comment (lnemsick-simp) [inline Rakefile]: I don't think we need this rake target anymore.  There is a ticket about this cruft.

File: README.md (hunk 1 above)
Comment (trevor-vaughan) [reply README.md]: @lnemsick-simp It depends on how you copy the files. I'm going with "this is not a bad idea to keep in writing".

**Pull request merged**